receiver/jaegerreceiver/config.go

```diff
@@ -78,8 +82,57 @@ type Config struct {
 }
 
 var _ config.Receiver = (*Config)(nil)
+var _ config.Unmarshable = (*Config)(nil)
 
 // Validate checks the receiver configuration is valid
 func (cfg *Config) Validate() error {
 	return nil
 }
+
+// Unmarshal a config.Parser into the config struct.
+func (cfg *Config) Unmarshal(componentParser *config.Parser) error {
+	if componentParser == nil || len(componentParser.AllKeys()) == 0 {
+		return fmt.Errorf("empty config for Jaeger receiver")
+	}
```

Comment:
For future PR, maybe this validation belongs to `Validate` not to unmarshal?

Reply:
:+1: I will have a look at the custom unmarshalers and move whatever makes sense to the validation method in a future PR